docs/kafka/getting-started-kafka/README.adoc

```diff
@@ -123,9 +123,14 @@ endif::[]
 * *Cloud region*: Select `US East, N. Virginia`.
 * *Availability zones*: Select `Multi`.
 ////
-. Use the slider to select the required instance size. Instances with more streaming units have more capacity. In {product-kafka}, you can create Kafka instances of 1 or 2 streaming units, however, instances larger than 1 streaming unit are currently available only as a Technology Preview and should not be used in production.
-. Click *Create instance* to start the creation process for your Kafka instance.
+. If you have purchased a subscription for {product-kafka}, you can use the slider to select an instance size. Instances with more streaming units have more capacity.
 +
+NOTE: In {product-kafka}, you can create Kafka instances of 1 or 2 streaming units, however, instances larger than 1 streaming unit are currently available only as a Technology Preview and should not be used in production.
```

Comment:
I think it _might_ improve readability to split this into two sentences, the second starting with "However,..."

I don't think this would be a big deal if it was in a standard paragraph. But this note is quite an important one, and should be very easy to parse.

Reply:
I wonder if we should consider an additional sentence to the note, along the lines of the following: "If you're creating a trial instance, the instance size is preconfigured."

Reply:
@jbyrne-redhat Yes. I can add that. I was thinking about adding something about trial instances but wasn't sure if this straying into @fmcdonal update on trial instances.

Reply:
@AedinC Well, maybe decide what is best between you! It just struck me that users might be confused about the absence of the slider if creating a trial instance.